src/RulesTest/PropertyConditions/BoolPropertiesTest.cs

```diff
@@ -315,5 +315,36 @@ public void TestIsControlElementDoesNotExistFalse()
                 Assert.IsFalse(IsControlElementDoesNotExist.Matches(e));
             } // using
         }
+
+        [TestMethod]
+        public void IsDialog_true()
```

Comment:
We've shifted the namign convention of these tests. If we use the existing convention, this might be
```suggestion
        public void IsDialogTrue()
```